Common/Enums/VehicleRole.sqx

```diff
@@ -33,5 +33,20 @@ namespace Intrusion.Common
 			
 			_sRole
 		};
+		
+		public static method fromString("_sRole" as String)
```

Comment:
Call it ToVehicleRole instead and add the return type:
"public static method VehicleRole ToVehicleRole("_sRole" as String)
Method named always with first letter capital.